modules/component-store/eslint.config.mjs

```diff
@@ -0,0 +1,54 @@
+import { FlatCompat } from '@eslint/eslintrc';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import js from '@eslint/js';
+import baseConfig from '../../eslint.config.mjs';
+
+const compat = new FlatCompat({
+  baseDirectory: dirname(fileURLToPath(import.meta.url)),
+  recommendedConfig: js.configs.recommended,
+});
+
+export default [
+  {
+    ignores: ['**/dist', '**/jest.config.ts', '**/schematics-core/test-setup.ts', '**/schematics-core/utility/standalone.ts'],
```

Comment:
Is it possible to add this to the base config and remove it in the module specific configs?

In the base config:

```ts
    ignores: ['**/dist', '**/jest.config.ts', '**/schematics-core/test-setup.ts', '**/schematics-core/utility/standalone.ts'],
```

Also, why do we need to exclude these files?

Reply:
All the exclusions were added to match previous files.  The lints fail without them.

Reply:
Potential option is to add `.eslintignore` files and add them there. 

In base config, could add an overrides array like this: 

```ts
  overrides: [
    {
      files: ['**/jest.config.ts'],
      env: { node: true },
      rules: {
        // customize rules for config files
      },
    },
    {
      files: ['**/schematics-core/**/*.ts'],
      env: { node: true },
      rules: {
        // loosen rules or add globals as needed
      },
    },
  ],
```

Typically, files are ignored if they use "globals" or unusual patterns that could break if linted.

Reply:
It's fine, we don't need to add a eslintignore - AFAIK this isn't supported anymore.